packages/features/troubleshooter/components/EventScheduleItem.tsx

```diff
@@ -4,19 +4,18 @@ import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { trpc } from "@calcom/trpc/react";
 import { Badge, Label } from "@calcom/ui";
 
-import { useTroubleshooterStore } from "../store";
 import { TroubleshooterListItemHeader } from "./TroubleshooterListItemContainer";
 
 export function EventScheduleItem() {
   const { t } = useLocale();
-  const selectedEventType = useTroubleshooterStore((state) => state.event);
-
-  const { data: schedule } = trpc.viewer.availability.schedule.getScheduleByEventSlug.useQuery(
+  const scheduleIdStr = new URLSearchParams(window.location.search).get("scheduleId");
+  const scheduleId = scheduleIdStr ? Number(scheduleIdStr) : null;
```

Comment:
can we use useRouter here?

Reply:
Sorry for the super late reply. I'm a beginner for web development and will keep working on this issue. 
I have tried useRouterQuery here to get the scheduleId.